libyul/backends/evm/EVMAssembly.cpp

```diff
@@ -43,31 +44,31 @@ void EVMAssembly::setSourceLocation(SourceLocation const&)
 	// Ignored for now;
 }
 
-void EVMAssembly::appendInstruction(solidity::Instruction _instr)
+void EVMAssembly::appendInstruction(dev::eth::Instruction _instr)
```

Comment:
`dev::eth` redundant here?

Reply:
This file is sometimes using `dev::eth` and sometimes not.

Reply:
Most of the places would clash with `yul::Instruction`.

Reply:
ah ok